org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/panel/InfoPanel.java

```diff
@@ -1318,21 +1320,23 @@ protected void validateOrderIndex() {
 		if (indexOrderColumn > 0 && (indexOrderColumn + 1 > p_layout.length || !p_layout[indexOrderColumn].getColSQL().trim().equals(sqlOrderColumn))) {
 			// try to find out new index of ordered column, in case has other column is hide or display
 			for (int testIndex = 0; testIndex < p_layout.length; testIndex++) {
-				if (p_layout[testIndex].getColSQL().trim().equals(sqlOrderColumn)) {
+				if (p_layout[testIndex].getColSQL().trim().equals(sqlOrderColumn) || sqlOrderColumn.equals(p_layout[testIndex].getDisplayColumn())) {
```

Comment:
can sqlOrderColumn be null here ?

Reply:
In the latest commit, I've modified ColumnInfo.getDisplayColumn() and these if statements to be NPE safe.